packages/web-components/src/components/link-list-section/__stories__/link-list-section.stories.ts

```diff
@@ -20,7 +20,8 @@ export const Default = ({ parameters }) => {
       <dds-link-list-heading>${heading}</dds-link-list-heading>
       <dds-link-list>
         <dds-link-list-item href="https://example.com">
-          Learn more about Kubernetes ${ArrowRight20({ slot: 'icon' })}
+          Learn more about Kubernetes, an open-source system for automating deployment, scaling, and management of containerized
+          applications. ${ArrowRight20({ slot: 'icon' })}
```

Comment:
I think we can shorten this to `Learn more about Kubernetes and automating deployment`